src/main/java/com/harrison/config/RabbitConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.harrison.config;
+
+import org.springframework.amqp.core.Queue;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+public class RabbitConfig {
+
+    @Bean
+    public Queue hello() {
+        Queue q = new Queue("hello", false, false, false);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "q". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS1488)